plugins/InspectionGadgets/src/inspectionDescriptions/MathRoundingWithIntArgument.html

```diff
@@ -0,0 +1,17 @@
+<html>
+<body>
+Reports calls to <code>round</code>, <code>ceil</code>,
```

Comment:
We try to always refer to methods with parenthesis included. That way it is not necessary to read the entire sentence to see that it refers to methods (or calls).
So, `Reports calls to <code>round()</code>..` etc. please.

Reply:
Thank you! I didn't know that. Changed